### PR TITLE
bring back provider.getStackName()

### PR DESCRIPTION
**Pull request by eahefnawy**
## What did you implement:

Closes #12345

## Todos:

- [ ] Write tests
- [ ] Write documentation
- [ ] Fix linting errors
- [ ] Make sure code coverage hasn't dropped
- [ ] Provide verification config/commands/resources
- [ ] Enable ["Allow edits from maintainers"](https://help.github.com/articles/allowing-changes-to-a-pull-request-branch-created-from-a-fork/) for this PR
- [ ] Change ready for review message below

***Is this ready for review?:*** NO

### CODE REVIEW

File: lib/plugins/aws/provider/awsProvider.js [hunk 1]
```diff
@@ -12,6 +12,21 @@ const constants = {
 };
 
 const impl = {
+
+  /**
+   * Deprecated method. Moved to naming.js
+   * Kept here for backward compatibility
+   */
+  getStackNmae(stage) {
```
Comment (pmuens) [inline lib/plugins/aws/provider/awsProvider.js]: `getStackName`

Comment (nikgraf) [inline lib/plugins/aws/provider/awsProvider.js]: @eahefnawy there is a typo: `Nmae` -> `Name`

wasn't it directly attached to the provided?

Comment (eahefnawy) [inline lib/plugins/aws/provider/awsProvider.js]: LOL nice!

**Comment by pmuens** [comment]
Looks good to me from a code perspective!

**Comment by nikgraf** [comment]
Looks good now 🎉

**Pull request merged**